README.md

```diff
@@ -63,14 +63,16 @@ DynamoDB tables are used to track the state and progress of a request. The follo
 
 ### File formats
 
-The DCP MS enables users to prepare expression matrices in several file formats by supplying the `format` parameter when
-submitting a POST request to the matrix endpoint. The following is a list of supported file formats:
+The DCP MS enables users to prepare expression matrices in several formats by supplying the `format` parameter in the
```

Comment:
is MS utilized everywhere in this readme? The acronym sounds a little odd just reading this line

Reply:
Yea it's introduced "Matrix Service (MS)" once and then I believe MS everywhere else. No problem, I'm fine with writing the full words in all cases

Reply:
Will address in a separate change

Reply:
And I planned to update to Expression Matrix Service in my README PR.

Reply:
Sure, I'll leave it to your PR then @brianraymor